site/en/docs/crux/release-notes/index.md

```diff
@@ -43,6 +43,21 @@ The CrUX dataset on BigQuery is generally updated on the second Tuesday of every
 
 In the list below, we've curated some release notes for each monthly dataset. Subscribe to our [CrUX Announce](https://groups.google.com/a/chromium.org/forum/#!forum/chrome-ux-report-announce) mailing list or follow [@ChromeUXReport](https://twitter.com/ChromeUXReport) on Twitter for release Announcements.
 
+## 202207
+
+[Announcement](https://groups.google.com/a/chromium.org/g/chrome-ux-report-announce/c/N5B3F9QVOmY)
+
+Publication date
+ : August 09, 2022
+
+What's new
+ : - This release includes a change which records the CLS metric at the first OnHidden in addition to tab close. See the [CLS changelog](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/2021_11_cls.md) for details.
```

Comment:
```suggestion
 : - This release includes a change which records the CLS metric at the first `OnHidden` in addition to tab close. See the [CLS changelog](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/2021_11_cls.md) for details.
```

Reply:
This _seems_ like it should be in a code font, but reject this if not.

Reply:
Let's reword it instead, as not sure how much of that is an internal status rather than a web standard one.

```suggestion
 : - This release includes a change which records the CLS metric when the tab is bacgrounded in addition to tab close. See the [CLS changelog](https://chromium.googlesource.com/chromium/src/+/refs/heads/main/docs/speed/metrics_changelog/2021_11_cls.md) for details.
```